code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -127,6 +127,7 @@
 	var/body_temperature = 310.15							// Species will try to stabilize at this temperature. (also affects temperature processing)
 
 	// Cold
+	var/hypothermia_difference = 17.5						// Hypothermia effects below cold_level_1 + this. 
```

Comment:
do you mean, below cold_level_1 in commine here?

Reply:
Uhh, confused by what you're asking. The comment is supposed to point out that hypothermia starts at a body temperature of cold_level_1 + this [variable], as in cold_level_1+hypothermia_difference